### PR TITLE
Add statistics to StyleCopTester

### PR DESCRIPTION
**Pull request by pdelvo**

### CODE REVIEW

**Comment by pdelvo** [comment]
I fixed the build break

File: StyleCop.Analyzers/StyleCopTester/Statistic.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    internal struct Statistic
+    {
+        public Statistic(int numberOf1, int numberOf2, int numberOf3)
+        {
+            this.NumberofNodes = numberOf1;
+            this.NumberOfTokens = numberOf2;
+            this.NumberOfTrivia = numberOf3;
+        }
+
+        public int NumberofNodes { get; set; }
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCopTester/Statistic.cs]: These properties don't need setters.

File: StyleCop.Analyzers/StyleCopTester/Statistic.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿namespace StyleCopTester
+{
+    internal struct Statistic
+    {
+        public Statistic(int numberOf1, int numberOf2, int numberOf3)
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCopTester/Statistic.cs]: :bulb: These parameter names could be... improved :expressionless:

**Comment by pdelvo** [comment]
I implemented the CR feedback

**Pull request merged**